packages/typespec-ts/test/modularUnit/scenarios/samples/parameters/paramaterSpreadCheck.md

```diff
@@ -0,0 +1,96 @@
+# skip: Should generate samples for spread cases
```

Comment:
Skipped due to the TCGC issue tracked: https://github.com/Azure/typespec-azure/issues/1421

Reply:
the tcgc isssue is resolve, I am not sure why we still skip this ?

Reply:
Also, I am not sure why the spread has relation with the serializeName and clientName?

Reply:
this tcgc issue will be released next week. we could enable this case at that time. 

because we have requiredHeader and optionalHeader property which will be renamed as required-header vs optional-header.